meta-mender-core/recipes-bsp/grub/grub-efi-mender-precompiled_2.04.bb

```diff
@@ -28,8 +28,6 @@ SRC_URI = " \
 
 S = "${WORKDIR}/git"
 
-require version_logic.inc
```

Comment:
The commit message says this is unnecessary, but doesn't expand on why. This specifies several dependencies and conflicts for GRUB, so why remove it?

Reply:
Just checked the code and the reason we removed it is that the whole file assumes grub is booted from u-boot and it's u-boot version conflicts with grub that is being searched for but we don't use u-boot so none of it helps.

Not really sure how to change it to work with both u-boot and edk2, any suggestions?